packages/block-editor/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 -   Removed unused `@wordpress/block-serialization-default-parser`, `css-mediaquery`, `memize` and `redux-multi` dependencies ([#38388](https://github.com/WordPress/gutenberg/pull/38388)).
 
+### New Features
+
+-   List View now supports selecting and dragging multiple blocks via `SHIFT` clicking items in the list [#38314](https://github.com/WordPress/gutenberg/pull/38314).
+>>>>>>> e0937d166f (Update documentation, add changelog entry)
```

Comment:
```suggestion
```

Reply:
Thanks! That wound up in there from a bad rebase 😀